src/core/Handler.js

```diff
@@ -21,6 +21,58 @@ export const Handler = Class.extend({
 
 		this._enabled = true;
 		this.addHooks();
+
+
+		function loadRepoData() {
```

Comment:
Wrong copy-paste?

Reply:
It is to late, i wanted to quickly add things before going to bed ... i copied the js code into this file to run `npm run lintfix` because it doesn't work in `plugins.md`. And then I forgot to remove it. I will fix it tomorrow